DEPENDENCIES

```diff
@@ -225,14 +225,16 @@ Prerequisites for building the OpenDataPlane (ODP) API
    $ meson build
    $ cd build
 
-   # Configure the location where DPDK will be installed
+   # Optionally, configure the location where DPDK will be installed. By default,
+   # DPDK will be installed in /usr/local:
    $ meson configure -Dprefix=$(pwd)/../install
 
    # Build and install DPDK
    $ ninja install
 
    # Configure ODP
-   # Instead of using --with-dpdk-path option, set PKG_CONFIG_PATH
+   $ ./configure --enable-dpdk
+   # Or, if DPDK was not installed at default location, set PKG_CONFIG_PATH:
```

Comment:
Maybe: `Or, if DPDK was not installed to the default location, set PKG_CONFIG_PATH:`

Reply:
The commit message could be simplified a bit:
```
In the current installation instructions for DPDK-20.11, configuring a
custom DPDK installation location is a required step. However, DPDK can
also be installed to the standard installation path.
```